src/helpers/Timer.hpp

```diff
@@ -6,7 +6,7 @@ class CTimer {
   public:
     void                                         reset();
     float                                        getSeconds();
-    long                                         getMillis();
+    float                                        getMillis();
```

Comment:
it'd be better to just make a new `getMicros` or `getMillisFloat`

Reply:
It is converted to float in every place it is used anyway. And all of those places benefit from better precision.